pkg/executables/kubectl.go

```diff
@@ -1162,3 +1163,34 @@ func (k *Kubectl) GetResource(ctx context.Context, resourceType string, name str
 	}
 	return found, err
 }
+
+func (k *Kubectl) getObject(ctx context.Context, resourceType, name, namespace, kubeconfig string, obj client.Object) error {
```

Comment:
Why cant we reuse the GetResource function?

Reply:
`GetResource` does something different: it returns true if the object exists and false if it doesn't